manifests/config.pp

```diff
@@ -124,7 +124,10 @@
 
     # End of cron based systemd
   } elsif $periodic_method == 'timer' {
-    $_updatedb_command = "/usr/bin/systemctl start ${locate}-updatedb.service"
+    $_updatedb_command = $facts['os']['family'] ? {
+      'Debian' => "/bin/systemctl start ${locate}-updatedb.service",
+      default  => "/usr/bin/systemctl start ${locate}-updatedb.service",
+    }
```

Comment:
The puppet-systemd module which we depend on does not provide the full path to `systemctl`, but the module use `exec` with a `path` to avoid hardcoding the full path, maybe we can do the same to avoid this:

https://github.com/voxpupuli/puppet-systemd/blob/4b01a5143a47f7ef51b3f4d29f9e5a5e3668c676/manifests/init.pp#L244-L248

Reply:
Re the PATH.

I've tried to change the systemd ones (and failed) since $facts.path changed between background and forground run which gives spurious cat' diffs.

Happy to change them here to hardcoded path?

Reply:
Am now using a static path of 

`/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`

since:

```
> podman run almalinux:9 env | grep PATH
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
> podman run debian:12 env | grep PATH
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
> podman run docker.io/archlinux env | grep PATH
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
so same for those 3.